crates/turbopack-ecmascript/src/parse.rs

```diff
@@ -250,7 +250,7 @@ async fn parse_content(
         Box::new(IssueEmitter {
             source,
             source_map: source_map.clone(),
-            title: Some("Parsing ecmascript source code failed".to_string()),
+            title: Some("Input ecmascript file had an error".to_string()),
```

Comment:
```suggestion
            title: Some("Ecmascript file had an error".to_string()),
```

It might be transformed, not sure if it's an "input" file.